eva-accession-import/README.md

```diff
@@ -0,0 +1,25 @@
+# EVA accession import
+
+This module can be used to import accessions (SS and RS) from a dbsnp mirror into the accessioning service at EVA. In https://github.com/EBIvariation/eva-accession/wiki there's some documentation about the transformations done.
+
+The program can be run with a plain `java -jar eva-accession-import-0.2.0-SNAPSHOT.jar` from a directory with an `application.properties` such as the one under `src/main/resources`.
+
+## Parameters
+
+Tips to fill some parameters:
+
+### `parameters.assemblyAccession`
+
+Look in the dbsnp mirror, in the contiginfo table, the columns "asm_acc" and "asm_version".
```

Comment:
`From the dbSNP mirror, concatenate columns 'asm_acc' and 'asm_version' in the 'contiginfo' table.`